### DISCUSSION TITLE
Do transactions need to be sorted?

### DISCUSSION THREAD

**Discussion by gbtorrance** [eprbell/rp2#88] [General]
Quick question: Within a particular asset sheet, do the IN, OUT, and INTRA transactions need to be sorted?

Of course it's ideal if they are, but does RP2 require this? (I'm asking because I've got some scripts to create the asset sheets from other sheets, and it's easiest if the sheets don't need to be sorted.)

Thanks.

**Comment by eprbell** [comment]
Hi Greg, no RP2 doesn't require input transactions to be sorted within the IN, OUT and INTRA tables. The RP2 output generator sorts them chronologically, but the user can re-sort them with different criteria using LibreOffice (or whatever spreadsheet software).

**Comment by gbtorrance** [comment]
Thanks for the info, @eprbell.
